functions/npm-diff.js

```diff
@@ -177,8 +177,9 @@ const diff = async ({ baseSha, baseRepoUrl, diffUrl, commentsUrl, token }) => {
   }
 }
 
-// eslint-disable-next-line max-statements
 const handler = async function (rawEvent) {
+  console.log(rawEvent)
```

Comment:
Did you mean to leave this here?

Reply:
It was taken from the original code:

https://github.com/netlify/plugins/pull/465/files#diff-667eea7fc09e29a1d494221528a5e822024a107e86f9f1f4557b3f3b146d7fb3L192

My guess is that this is meant for production debugging.
However, considering most of the code has error handling setup, should this be removed? What do you think?

Reply:
Oh, I see. I'm good either way, so whatever you feel is best.

Reply:
Let's keep it for the moment :+1: